erpnext/controllers/accounts_controller.py

```diff
@@ -744,8 +744,9 @@ def validate_multiple_billing(self, ref_dt, item_ref_dn, based_on, parentfield):
 					role_allowed_to_over_bill = frappe.db.get_single_value('Accounts Settings', 'role_allowed_to_over_bill')
 
 					if total_billed_amt - max_allowed_amt > 0.01 and role_allowed_to_over_bill not in frappe.get_roles():
-						frappe.throw(_("Cannot overbill for Item {0} in row {1} more than {2}. To allow over-billing, please set allowance in Accounts Settings")
-							.format(item.item_code, item.idx, max_allowed_amt))
+						if not cint(frappe.db.get_single_value("Buying Settings", "consider_rejected_quantity_in_purchase_invoice")):
```

Comment:
Bypass overbilling only for Purchase Invoice

Reply:
Added